xacro/cli.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3
```

Comment:
I don't think the shebang line is necessary since this isn't intended to be executed as a script.